### PR TITLE
Add Event Roles column in admin/users

### LINKED ISSUE #1298: The Event Role Column in admin/users displays nothing.
**Describe the bug**

The column in event role in admin/users currently displays nothing.

**To Reproduce**
1. Go to 'admin/users'
2. Check the column Event Role
3. Nothing is shown

**Expected behavior**
It should show the events of which the user is the organizer, coorganizer, track organizer, moderator, registrar of

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: Chromium

### PR DESCRIPTION
**Pull request by ritikamotwani**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
It adds information to event role column in admin/users table

#### Changes proposed in this pull request:

- Adds relevant information to the column event roles
image

Fixes #1298

### CODE REVIEW

File: app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs [hunk 1]
```diff
@@ -1,5 +1,55 @@
-<div class="ui ordered list">
-  {{#each record.eventRoles as |eventRole|}}
-    <div class="item">{{eventRole.name}} ({{eventRole.event.name}})</div>
+{{#if record.isUserOrganizer}}
+  {{#each record.events as |event|}}
```
Comment (niranjan94) [inline app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs]: This is not the right way to do this. You're basically looping through all events that user is a part of. Whether he is an organizer or not. Then you're looping through all the organizers of that event. And then you're matching the email address. So many requests and loops.

File: app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs [hunk 2]
```diff
@@ -1,5 +1,55 @@
-<div class="ui ordered list">
-  {{#each record.eventRoles as |eventRole|}}
-    <div class="item">{{eventRole.name}} ({{eventRole.event.name}})</div>
+{{#if record.isUserOrganizer}}
```
Comment (niranjan94) [inline app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs]: This is not the right way to do this. You're basically looping through all events that user is a part of. Whether he is an organizer or not. Then you're looping through all the organizers of that event. And then you're matching the email address. So many requests and loops.

Couldn't you just use the `coorganizer-events`, `moderator-events`, `track-organizer-events`, `organizer-events`, etc relations of the user to directly get the events that the user is a part of. ? Also, make sure you request _only_ what you need. Here you don't need the full event info. You only need event identifier and event name. Then please request and use just that.

**Comment by ritikamotwani** [comment]
@niranjan94 in the ui template I have a doubt. Like I am currently printing event.id, it gives the correct event ids of the events for which the user is an organizer or  co organizer. But if I try to print event.name/identifier or other information nothing is printed. Am I missing out something?

File: app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs [hunk 3]
```diff
@@ -1,5 +1,17 @@
 <div class="ui ordered list">
-  {{#each record.eventRoles as |eventRole|}}
-    <div class="item">{{eventRole.name}} ({{eventRole.event.name}})</div>
+  {{#each record.organizerEvents as |event|}}
+    <div class="item">{{event.id}}{{t ' - Organizer'}}</div>
```
Comment (ritikamotwani) [inline app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs]: @niranjan94  I have a doubt here. Like I am currently printing event.id, it gives the correct ids of the events for which the user is an organizer or co organizer. But if I try to print event.name/identifier or other information nothing is printed. Am I missing out something?

**Comment by bhaveshAn** [comment]
@niranjan94 Merged it. As it was needed as priority. In case you find anything wrong, you can follow up by creating an issue or in the channel and then the suggestion can be followed up by the dev. Hope you don't mind.
Thanks !!

**Pull request merged**